wrapanapi/systems/lenovo.py

```diff
@@ -90,6 +90,15 @@ def _service_put(self, path, request):
         except Timeout:
             return None
 
+    def _service_post(self, path, request):
+        """Makes POST request and returns the response"""
+        try:
+            response = requests.post('{}/{}'.format(self.url, path), data=json.dumps(request),
```

Comment:
that was quick! :)

Reply:
haha! just making sure this will be merged asap :D